docs/guides/chip_tool_guide.md

```diff
@@ -778,25 +853,110 @@ following command:
+    [Commissioning into a Thread network over Bluetooth LE](#commissioning-into-a-thread-network-over-bluetooth-le).
+-   `ble-wifi` - For Wi-Fi commissioning; described under
+    [Commissioning into a Wi-Fi network over Bluetooth LE](#commissioning-into-a-wi-fi-network-over-bluetooth-le)
+-   `onnetwork` - For commissioning the device when it is already present in an
+    IP network; described under
+    [Commissioning with setup PIN code](#commissioning-with-setup-pin-code)
+
+To list all `pairing` sub-commands and commissioning methods, run the following
+command:
 
 ```
 $ ./chip-tool pairing
 ```
 
+**Example of commands:**
+
+The following command commissions the Thread device with the node ID `1` to the
+Matter fabric. The `hex:...` parameter is the operational dataset that contains
+information about the Thread network to which the device is going to be
+commissioned. The onboarding dataset payload `34970112332` (short manual pairing
+code) is used to discover and commission the device.
+
+```
+$ ./chip-tool pairing code-thread 1 hex:000030000150208562618342348532605109bd31cda6908667addca8789211addac0102c4a9 34970112332
+```
+
+The following command commissions the Wi-Fi device with the node ID `1` to the
+Matter fabric The SSID `wifi_test` and the password `admin123` is the required
+information about the Wi-Fi network to which the device is going to be
+commissioned. The onboarding dataset payload `34970112332` (short manual pairing
+code) is used to discover and commission the device.
+
+```
+$ ./chip-tool pairing code-wifi 1 wifi_test admin123 34970112332
+```
+
+The following command commissions the device with the node ID `1` to the Matter
+fabric. The onboarding dataset payload `MT:8IXS142C00KA0648G00` (QR code
+payload) is used to discover and commission the device.
+
+```
+$ ./chip-tool pairing code 1 MT:8IXS142C00KA0648G00
+```
+
+#### Attestation-related flags
+
+The `pairing` commissioning command can be run with several flags that allow you
+to modify attestation-related settings:
+
+-   `--paa-trust-store-path` - Use to provide the path to the directory that
+    contains the information about Product Attestation Authority (PAA)
+    certificates. The path can be absolute or relative to the current working
+    directory. With this flag, the CHIP Tool looks for the PAA certificate that
+    matches the PAI and the DAC certificates programmed on the device. Without
+    this flag, the CHIP Tool uses the built-in test PAA certificate.
+
+-   `--cd-trust-store-path` - Use to provide the path to the directory
+    containing the key that is used to generate the Certification Declaration.
+    The path can be absolute or relative to the current working directory. With
+    this flag, the CHIP tool looks for the public key that successfully validate
+    device Certification Declaration's signature. Without this flag, the CHIP
+    Tool uses the built-in test signing key.
```

Comment:
I don't think this matches actual behavior.  Did you speak with someone "very familiar with how CD works" as I asked?  Because as far as I can tell default behavior uses keys other than the test signing key....